recipe/bld.bat

```diff
@@ -1,4 +1,6 @@
 
+git clone --depth 1 https://github.com/davisking/dlib.git dlib_git && cd dlib_git
```

Comment:
What's this doing here?...

There's `meta.yaml` for pulling source, at specific git revisions / tags / branches.

Reply:
I was just experimenting, I'll remove it, sorry